content/en/docs/reference/controllers/statefulset.md

```diff
@@ -0,0 +1,89 @@
+---
+title: StatefulSet controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The StatefulSet controller ensures that a {{< glossary_tooltip term_id="StatefulSet" >}}
+is running on a suitable set of Nodes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+The StatefulSet controller is built in to kube-controller-manager.
+
+## Controller behavior
+
+The controller creates a number of Pods for each StatefulSet that it observes,
+based on the count of replicas configured for that StatefulSet. The controller
+creates these Pods *sequentially*.
+
+When the StatefulSet controller creates a Pod, it adds a label, `statefulset.kubernetes.io/pod-name`,
+that is set to the name of the Pod. This label gives each Pod a durable, stable
+network identity, that allows you to attach a Service to a specific Pod in
+the StatefulSet.
+The controller constructs the pod name label valuebased on a pattern:
+`$(statefulset name)-$(ordinal)`.
+
+### Storage
```

Comment:
Similar to Deployment and Job, I think much of the remainder of this page should be in a concepts/workloads/statefulset.md document that is written from an end-user or object reference perspective. And leave this doc for implementation specifics about the controller itself.